packages/cli/src/commands/source/push/add.spec.ts

```diff
@@ -42,15 +38,15 @@ describe('source:push:add', () => {
     mockEvaluate.mockResolvedValue({approved: false});
   };
 
-  const mockConfig = () => {
-    mockedConfigGet.mockReturnValue({
-      region: 'us',
-      organization: 'foo',
-      environment: 'prod',
-    });
+  // const mockConfig = () => {
+  //   mockedConfigGet.mockReturnValue({
+  //     region: 'au',
+  //     organization: 'foo',
+  //     environment: 'prod',
+  //   });
 
-    mockedConfig.prototype.get = mockedConfigGet;
-  };
+  //   mockedConfig.prototype.get = mockedConfigGet;
+  // };
```

Comment:
?

Reply:
oups. sorry to remove that.
We no longer need to mock config, since  we are fetching the org/region and env from the authenticated client (which is already moked)